.travis.yml

```diff
@@ -1,37 +1,61 @@
-notifications:
-  email: false
-
-sudo: false
-
-os:
-  - osx
-  - linux
-
-addons:
-  apt:
-    packages:
-      - xclip
-
-language: node_js
-
-node_js:
-  - "6.3.1"
-
-env:
-  - GITHUB_TOKEN=1b17d62d38a4846efa7ea4de4b773b581787b0f1
-
-before_install:
-  - if [ $TRAVIS_OS_NAME == "linux" ]; then
-      export CXX="g++-4.9" CC="gcc-4.9" DISPLAY=:99.0;
-      sh -e /etc/init.d/xvfb start;
-      sleep 3;
-    fi
-
-install:
-  - npm install
-  - npm run vscode:prepublish
-
-script:
-  - npm install -g gulp
-  - gulp
-  - npm test --silent
+notifications:
+  email: false
+
+sudo: false
+
+os:
+- osx
+- linux
+
+addons:
+  apt:
+    packages:
+    - xclip
+
+language: node_js
+
+node_js:
+- 6.3.1
+
+env:
+  global:
+    secure: F3+6ObO8G2XDTXXSWizxdV4YzjWWwFKfISwYnKTwnAy4LXLPzcHnrh31rFmKeXSddZmWiVMhy++a4Y9IOTi0DrSnUYaaIP35zMIvMYIqdSa6v3b93Z0fwZmniXe0e4EBIhcdnMOz1fdxGhd0oWP/YyywOm32TjymhqbApT9AzqUodpGqRmA30B1tOFr09K+A2EiIr+4QGsB/a3VUv24zWWUHtkXaJ6tQ7rqt6944tGK4RtjAIzXLoePjJ+kmTqY6fMk/sPmYxHXnzjaEsIFBd5dYDCFtjR0oqjdq2OXc+bwIjl4t3Wh4RwE8bUEcUSnkPRqlJH5Hni5vfPRirb4qH4uoFmLNa8Pg2dPI44cYBvWVGSJ5GqXD5jV64waR3CM3VC2Ll76KWdlqdUo0cNIV75ul+UdieuNghTnw3OTYIWyhvrLqfnabOG81z09brNapZuCPj3rZ+sjTWf7beRoH877UGH0C6jimxNk0syjGL0+pXH0UR0dI+7RWaNyPAMfvjz3m9UpDw3NtdDGNMT1D3VgZz+INwU6XMUSpg36S1BzMTfi6NJm+srA5BDHyC/MuYjO+l/1gaj0+dvkpMyyj9btotYFmfemEFlm2AHGMkITaucUrmCQKIM9ZL7l/fZfvPkUGbaaLH7EONL7KLLaj6pk2z7na91/3I1pWg4yJDPE=
```

Comment:
I think they would have the same effect.

I would prefer putting it in this manner as then our configs are in one central location and not scattered about.